src/main/resources/application.yaml

```diff
@@ -21,6 +21,13 @@ damap:
   projects-url: http://api-mock:80
   persons-url: http://api-mock:80
```

Comment:
possibly antiquated solution for conveying api address, considering that we want to now support n-solutions.
can be removed, or need to convey a new pattern that forces adding this property.

Reply:
was moved to its own issue GH-65